src/main/java/taxi/dao/DriverDaoImpl.java

```diff
@@ -17,13 +17,15 @@
 public class DriverDaoImpl implements DriverDao {
     @Override
     public Driver create(Driver driver) {
-        String query = "INSERT INTO drivers (name, license_number) "
-                + "VALUES (?, ?)";
+        String query = "INSERT INTO drivers (name, license_number,login,password) "
+                + "VALUES (?, ?,?,?)";
```

Comment:
It's better to keep consistency in code even if it's spaces. Will look more tidy